### ISSUE TITLE
[Accessibility] High contrast issues

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt#2505]
Focus: When in high contrast mode, it's unclear when tabbing that the more and help menu items are expanded / focused: 

![screen shot 2017-07-24 at 2 18 37 pm](https://user-images.githubusercontent.com/16690124/28545230-0e7847a8-707b-11e7-83fe-a348d7cb7abe.png)

Cookie dialog: 
![screen shot 2017-07-24 at 2 58 01 pm](https://user-images.githubusercontent.com/16690124/28546718-98d03b86-7080-11e7-8647-b427eca5d285.png)

Error message: 
![screen shot 2017-07-24 at 2 58 54 pm](https://user-images.githubusercontent.com/16690124/28546734-a6401246-7080-11e7-8dec-ffeebd31539f.png)

Full screen simulator background should always be black. 

Make monaco toolbox icons white.

**Comment by veler** [comment]
Issues fixed in this PR : [https://github.com/Microsoft/pxt/pull/2507](https://github.com/Microsoft/pxt/pull/2507)
